src/System.Security.Permissions/tests/System.Security.Permissions.Tests.csproj

```diff
@@ -26,6 +26,7 @@
     <Compile Include="SecurityElementTests.cs" />
     <Compile Include="HostSecurityManagerTests.cs" />
     <Compile Include="HostProtectionTests.cs" />
+    <Compile Include="SecurityManagerTests.cs" />
```

Comment:
I know this isn't related to your change but please sort the files as we were supposed to do before we got lazy ;)